### ISSUE TITLE
ترجمة التطبيق إلى اللغة العربية / translate app into arabic

### ISSUE THREAD

**Issue by 0xadada** [mirai-audio/mir#48]
### Expected Behavior

Arabic speakers can use the application in mandarin chinese language.

### Task

All translations are located in https://github.com/mirai-audio/mir/tree/master/app/locales, with directory for each translation language. 

These should be added under a new `ar` directory in `app/locales` according to the `ISO 639-1` standard names.

Please use English as the source for translations, as it contains a complete list of up-to-date strings.

Please use our [Writing Styleguide](https://github.com/mirai-audio/mir/wiki/Writing-Styleguide) for how the voice & persona of the text should come across.

**Comment by Aminelas** [comment]
Hello, 
I would like to add an arabic translation for the app.

**Comment by 0xadada** [comment]
@aminelas yes, please do!
